app/models/agents/http_agent.rb

```diff
@@ -14,9 +14,9 @@ class HttpStatusAgent < Agent
     form_configurable :disable_redirect_follow, type: :array, values: ['true', 'false']
 
     description <<-MD
-      The HttpStatusAgent will check a url and emit the resulting HTTP status code.
+      The HttpAgent will check a url and emit the resulting HTTP status code and, optionally, the value of a specified header.
```

Comment:
Maybe, instead of only returning one, it should emit all returned headers if a `include_headers` flag is set to `true`?  Alternatively, you could allow setting a `headers` option that can be a single header name, or an array of header names, to emit.